Automattic/wp-calypso
# SPT: Make page title visibility depend on theme setting

#### Changes proposed in this Pull Request

Originated from https://github.com/Automattic/wp-calypso/issues/37509#issuecomment-591402834.

In the preview of a starter page templates, check the `Hide Homepage Title` theme option and set the title visibility accordingly.

Customizer setting `off`:

<img width="1143" alt="Screenshot 2020-02-26 14 51 07" src="https://user-images.githubusercontent.com/1451471/75351434-7346ef00-58a8-11ea-8c35-aff355000df1.png">

Customizer setting `on`:

<img width="1140" alt="Screenshot 2020-02-26 14 50 58" src="https://user-images.githubusercontent.com/1451471/75351483-848ffb80-58a8-11ea-8c19-b45c518483ad.png">

Page preview with the setting `off`:

<img width="1153" alt="Screenshot 2020-02-26 14 55 25" src="https://user-images.githubusercontent.com/1451471/75351543-9bcee900-58a8-11ea-94ab-b869f114117c.png">

Page preview with the setting `on`:

<img width="1154" alt="Screenshot 2020-02-26 14 52 08" src="https://user-images.githubusercontent.com/1451471/75351533-97a2cb80-58a8-11ea-9c93-3820ef005aa6.png">

#### Testing instructions

1. In Customizer, tick the `Homepage Settings / Hide Homepage Title` option checkbox,
2. Add a new page to your site - the Starter Page Templates picker should open,
3. Click any template - the preview window should not contain the page title.
